app/public/javascripts/calendar.js

```diff
@@ -28,6 +27,19 @@ var Calendar = function() {
 		wrap.find("#next").bind("click.calendar", function() {
 		 switchMonth(true);
 		});
+		//adds event listener to dayviews
```

Comment:
Instead of inlining this call, you should consider extracting this to another named function, thereby making the code more readable.